### PR TITLE
Center login page and show success and error message on login / forgot password

### LINKED ISSUE #46: Align the login form / forgot password at the center of the screen

### PR DESCRIPTION
**Pull request by ghost**
@iguit0 , I have also changed the logo from the application and fixed the bug when installing the dependencies (see package.json, the update for React Scrips), resolve #46 #49

### CODE REVIEW

**Comment by ghost** [comment]
[Reference bug React Scripts](https://github.com/ionic-team/ionic/issues/20503#issuecomment-586743378)

**Comment by ghost** [comment]
[Reference context level Noti Snack](https://github.com/iamhosseindhv/notistack/issues/145#issuecomment-510926701)

File: src/pages/login/login.js [hunk 1]
```diff
@@ -133,6 +152,11 @@ const FormikSign = ({ classes, setAuthentication, setUser, props }) => (
         resetForm();
       } catch (error) {
         console.log(error);
+
+        enqueueSnackbar("Error on login, contact an admin", {
```
Comment (iguit0) [inline src/pages/login/login.js:156]: @Sphinxs This message should me more clear to the user.
"contact an admin" sounds like an internal error, which is not true.
It's just the password (or e-mail) that are incorrect.

**Review by iguit0 — CHANGES_REQUESTED** [review]
1) Images are broken and not showing properly.
logo_img_broken

2) Successful message (on login) are not showing up.

@Sphinxs

File: src/pages/login/login.js (hunk 1 above)
Comment (ghost) [reply src/pages/login/login.js:156]: Yep, I have removed the contact admin into another PR

**Comment by ghost** [comment]
> 1. Images are broken and not showing properly.
>    logo_img_broken
> 2. Successful message (on login) are not showing up.
> 
> @Sphinxs

The image will be fixed at the #48 , once I need to download a image editor and the message is not shown because the user is redirected

**Review by iguit0 — APPROVED** [review]

**Pull request merged**